subprocess/redirects.go

```diff
@@ -18,7 +18,7 @@ type Redirect struct {
 	MaxOutputSize int64
 }
 
-const MAX_MEM_OUTPUT = 1024 * 1024
+const MAX_MEM_OUTPUT = 1024 * 1024 * 1024
```

Comment:
I'll probably follow up later today to make different constants for memory and disk output - I don't want to make 1G the limit for memory output as well.